api/src/modules/content-pages/queries/avo/updateContentPagePublishDates.graphql

```diff
@@ -9,7 +9,7 @@ mutation updateContentPagePublishDates($now: timestamptz, $publishedAt: timestam
 				{ publish_at: { _lte: $now, _is_null: false }, depublish_at: { _is_null: true } }
 				{ publish_at: { _is_null: true }, published_at: { _gte: $now, _is_null: false } }
 			]
-			published_at: { _is_null: true }
+			is_public: { _eq: false }
```

Comment:
Maybe even safer, since is_public should always be in sync with published_at

published_at is the date
is_public is a boolean

```suggestion
			_or: {
				is_public: { _eq: false },
				published_at: { _is_null: true }
			},
```

is_public should actually be a computed field of published_at, but maybe that's too much to change for a bug fix